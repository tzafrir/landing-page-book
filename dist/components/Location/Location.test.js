'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _enzymeAdapterReact = require('enzyme-adapter-react-16');

var _enzymeAdapterReact2 = _interopRequireDefault(_enzymeAdapterReact);

var _Location = require('./Location');

var _Location2 = _interopRequireDefault(_Location);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

(0, _enzyme.configure)({ adapter: new _enzymeAdapterReact2.default() });

describe('Location', function () {
  var props = void 0;
  var mounted = void 0;
  var location = function location() {
    if (!mounted) {
      mounted = (0, _enzyme.shallow)(_react2.default.createElement(_Location2.default, props));
    }
    return mounted;
  };
  describe('Location', function () {
    beforeEach(function () {
      props = {};
      mounted = undefined;
    });

    it('should render', function () {
      expect((0, _enzyme.shallow)(_react2.default.createElement(_Location2.default, props))).toMatchSnapshot();
    });

    it('always renders a div', function () {
      var divs = location().find('div');

      expect(divs.length).toBe(0);
    });
  });
});