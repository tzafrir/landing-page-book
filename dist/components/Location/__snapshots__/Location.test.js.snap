// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location Location should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Location
    apiKey=""
    language="en"
    lat={0}
    lng={0}
    markers={Array []}
    zoom={1}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "bootstrapURLKeys": Object {
        "key": "",
        "language": "en",
      },
      "center": Object {
        "lat": 0,
        "lng": 0,
      },
      "children": Array [],
      "debounced": true,
      "distanceToMouse": [Function],
      "googleMapLoader": [Function],
      "heatmap": Object {},
      "heatmapLibrary": false,
      "hoverDistance": 30,
      "layerTypes": Array [],
      "options": [Function],
      "style": Object {
        "height": "100%",
        "margin": 0,
        "padding": 0,
        "position": "relative",
        "width": "100%",
      },
      "yesIWantToUseGoogleMapApiInternals": false,
      "zoom": 1,
    },
    "ref": null,
    "rendered": Array [],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bootstrapURLKeys": Object {
          "key": "",
          "language": "en",
        },
        "center": Object {
          "lat": 0,
          "lng": 0,
        },
        "children": Array [],
        "debounced": true,
        "distanceToMouse": [Function],
        "googleMapLoader": [Function],
        "heatmap": Object {},
        "heatmapLibrary": false,
        "hoverDistance": 30,
        "layerTypes": Array [],
        "options": [Function],
        "style": Object {
          "height": "100%",
          "margin": 0,
          "padding": 0,
          "position": "relative",
          "width": "100%",
        },
        "yesIWantToUseGoogleMapApiInternals": false,
        "zoom": 1,
      },
      "ref": null,
      "rendered": Array [],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
